tox.ini

```diff
@@ -20,13 +21,15 @@ commands =
 
 [testenv:type]
 skipsdist = true
```

Comment:
```suggestion
```

I'd remove `skipsdist` too. The distinction between it and `skip_install` isn't super clear (as it needs to make an sdist to do the install, but I practically think of them as coupled Tox settings.